tunnel/tunnel.go

```diff
@@ -257,17 +182,37 @@ func createTunnel(tunnelConfig *Config) (net.Listener, error) {
 	// Listen on remote server port
 	listener, err = sClient.Listen("tcp", remoteEndpoint.String())
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "listen open port ON remote server error: %s", err)
+		fmt.Fprintf(os.Stderr, "listen open port ON remote server error: %s\n", err)
```

Comment:
Is this a useful error for the user?  Should we use log.Errorf here or is this standard feedback?

Reply:
This should be more clear